lms/djangoapps/teams/tests/test_views.py

```diff
@@ -147,6 +148,57 @@ def test_bad_course_id(self):
         response = self.client.get(bad_team_url)
         self.assertEqual(404, response.status_code)
 
+    def get_user_course_specific_teams_list(self):
+        """Gets the list of user course specific teams."""
+
+        def get_team_count_from_response(content):
+            """Gets teams count from response."""
+            try:
+                team_data = re.search('"teams": (.+?), "privileged"', content).group(1)
+            except AttributeError:
+                team_data = ''
+            return json.loads(team_data)['count'] if team_data else 0
```

Comment:
Just because adding this function adds complexity and is a bit of overkill for the test